docs/installation.rst

```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
 
   pip install "modin[dask]" # If you want to use the Dask backend
 
+Installing with conda
+---------------------
+
+Modin releases can be installed using ``Miniconda`` from conda-forge channel. Starting from 0.10.1
```

Comment:
```suggestion
Modin releases can be installed using ``conda`` from conda-forge channel. Starting from 0.10.1
```
I don't see a point of referencing `miniconda` specifically, that's just a distribution while we're speaking about the tool.